app/src/main/java/com/github/se/project/ui/overview/Overview.kt

```diff
@@ -287,46 +304,80 @@ private fun ExpandableLessonSection(
     onClick: (Lesson) -> Unit,
     listProfilesViewModel: ListProfilesViewModel
 ) {
-  var expanded by remember { mutableStateOf(lessons.isNotEmpty()) }
+    val isInstant = lessons.any { isInstant(it) }
+    var expanded by remember { mutableStateOf(if (isInstant) true else lessons.isNotEmpty()) }
+
+    val infiniteTransition = rememberInfiniteTransition(label = "iconBlink")
+    val alpha by infiniteTransition.animateFloat(
+        initialValue = 1f,
+        targetValue = 0.3f,
+        animationSpec = infiniteRepeatable(
+            animation = tween(500),
+            repeatMode = RepeatMode.Reverse
+        ),
+        label = "blinkAlpha"
+    )
 
-  Card(
-      modifier = Modifier.fillMaxWidth().testTag("section_${section.title}"),
-      colors =
-          CardDefaults.cardColors(
-              containerColor = MaterialTheme.colorScheme.surfaceContainerLowest)) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .testTag("section_${section.title}"),
+        colors = CardDefaults.cardColors(
+            containerColor = if (isInstant)
+                MaterialTheme.colorScheme.surfaceBright.copy(alpha = 0.7f)
+            else
+                MaterialTheme.colorScheme.surfaceContainerLowest
+        )
+    ) {
         Column {
-          ListItem(
-              headlineContent = {
-                Text(section.title, style = MaterialTheme.typography.titleMedium)
-              },
-              colors =
-                  ListItemDefaults.colors(
-                      containerColor = MaterialTheme.colorScheme.surfaceContainerLowest),
-              leadingContent = {
-                Icon(section.icon, null, tint = MaterialTheme.colorScheme.primary)
-              },
-              trailingContent = {
-                IconButton(onClick = { expanded = !expanded }) {
-                  Icon(
-                      if (expanded) Icons.Default.KeyboardArrowDown
-                      else Icons.Default.KeyboardArrowLeft,
-                      contentDescription = if (expanded) "Collapse" else "Expand")
-                }
-              },
-              modifier = Modifier.clickable { expanded = !expanded })
+            ListItem(
+                headlineContent = {
+                    Text(section.title, style = MaterialTheme.typography.titleMedium)
```

Comment:
I think somme of those changed lines are due to the missing KTFMT format, pleaaaase don't forget it as it would make it easier to review (and actually not crash the CI)! :)